desktopVersion.css

```diff
@@ -0,0 +1,502 @@
+    text-align: center;
+    background: transparent;
+    line-height: 24px;
+    border: none;
+    font-size: 17px;
+    letter-spacing: 0.03em;
+    cursor: pointer;
+  }
+
+  .lang__header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .lang__header h2 {
+    font-family: "Poppins", sans-serif;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #344563;
+  }
+
+  .lang__container {
+    display: flex;
+    gap: 1rem;
+    margin-bottom: 2rem;
+    padding-right: 1rem;
+    margin-top: 2rem;
+  }
+
+  .lang__container__type {
+    display: flex;
+    flex-direction: column;
+    background-color: #f7f7f9;
+    border-radius: 8px;
+    width: 80px;
+    height: 80px;
+    padding: 12px;
+  }
+
+  .lang__container__type img {
+    width: 34px;
+    height: 34px;
+  }
+
+  .lang__container__type span {
+    font-family: "Poppins", sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: 0.4rem;
+  }
+
+  .framework {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 2rem;
+    padding-right: 1rem;
+    border-bottom: 1.4px solid #f9f7f7;
+  }
+
+  .arrow {
+    color: #f7f7f9;
+    height: 22px;
+    width: 12px;
+  }
+
+  /* ========================CONTACT ME=========================== */
+
+  .contact__container {
+    background-image: url(img/contact\ form\ background\ shapes.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-color: #6070ff;
+    border-top-left-radius: 6rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 50px 250px;
+  }
+
+  form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1rem;
+  }
+
+  .contact__container h1 {
+    font-family: "Poppins", sans-serif;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    letter-spacing: 0;
+    text-align: center;
+    color: #fff;
+  }
+
+  .contact__container p {
+    font-family: "Poppins", sans-serif;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 28px;
+    letter-spacing: 0;
+    text-align: center;
+    color: #ebebff;
+    margin-bottom: 2rem;
+  }
+
+  input {
+    width: 100%;
+    padding: 0 15px;
+    border: none;
+    outline: none;
+    height: 40px;
+  }
+
+  textarea {
+    font-family: 'Poppins', sans-serif;
+    width: 100%;
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    padding: 15px;
+    border: none;
+    outline: none;
+    resize: none;
+  }
+
+  .get__in {
+    border-radius: 8px;
+    color: #396df2;
+    background-color: white;
+    width: 120px;
+    height: 40px;
+    text-align: center;
+    margin-bottom: 2rem;
+    font-size: 17px;
+    outline: none;
+    border: none;
+  }
+}
```

Comment:
- [ ] As mentioned by the previous reviewer, you're required to stick to the figma design as close as possible. Your contact me section is quite different from the figma design. Kindly compare the two screenshots below and walk your design alongside the figma design

### Your Design
![your-form](https://github.com/sibomana123/MV__Portfolio/assets/26597432/a2f80841-6ef2-4055-afd8-9fd0037e4961)

### Figam Design
![temp-form](https://github.com/sibomana123/MV__Portfolio/assets/26597432/64d8cc7b-3093-4179-acdb-fd3497437a0b)